tests/integration/reqs/test_ledger.py

```diff
@@ -1,5 +1,6 @@
 from tests.integration.integration_test_case import IntegrationTestCase
 from tests.integration.it_utils import test_async_and_sync
+from xrpl.asyncio.ledger import get_latest_open_ledger_sequence
```

Comment:
This function should be tested in `tests/integration/sugar/test_ledger.py`. This file is for testing the `ledger` RPC.

Reply:
updated in 25e7304c